src/universal/components/CreateCard/CreateCard.js

```diff
@@ -6,18 +6,20 @@ import Ellipsis from '../Ellipsis/Ellipsis';
 import Type from '../Type/Type';
 import theme from 'universal/styles/theme';
 import withHotkey from 'react-hotkey-hoc';
+import CreateCardRootStyles from './CreateCardRootStyles';
 
 const combineStyles = StyleSheet.combineStyles;
```

Comment:
The base styles for the `CreateCard` got copied in a couple of places, so I wanted them to share those same styles. So I import the object in a few components.